duneanaobj/StandardRecord/SRBeamBranch.h

```diff
@@ -27,7 +28,17 @@ namespace caf
       unsigned long int    gpspulsetimensec    = 0;         ///< pulse time from GPS [ns]
       signed long long int deltapulsetimensec  = -9999999;  ///< Delta time [ns]
       float                pulsepot            = NaN;       ///< POT in pulse including factor of 1e12 so that a user does not have to apply this correction
+      float                potTOR101           = NaN;       ///< POT as measured by the toroid E:TOR101
+      float                potTR101D           = NaN;       ///< POT as measured by the toroid E:TR101D
       float                hornI               = NaN;       ///< Horn current [kA]
+      float                hornDir             = NaN;       ///< Horn polarity 
+      std::vector<double>  horizontalposTGT;                ///< horizontal beam position as measured by E:HPTGT beam monitor 
+      std::vector<double>  horizontalintTGT;                ///< horizontal beam intensity as measured by E:HITGT beam monitor 
+      std::vector<double>  horizontalpos121;                ///< horizontal beam position as measured by E:HP121 beam monitor 
+      std::vector<double>  verticalposTGT;                  ///< vertical beam position as measured by E:VPTGT beam monitor 
+      std::vector<double>  verticalintTGT;                  ///< vertical beam intensity as measured by E:VITGT beam monitor 
+      std::vector<double>  verticalpos121;                  ///< vertical beam position as measured by E:VP121 beam monitor 
+      std::vector<double>  multiwireInfo;                   ///< measured beam intensity from horizontal and vertical wires  from E:MTGTDS beam monitor
```

Comment:
Could you ensure the comments here explain why these are vectors and not just numbers?  Also, it would be helpful if there were a comment at the beginning of this block linking to the explanations of the various beam device labels so that folks in the future can figure out what these are / how to use them